ci_framework/roles/devscripts/tasks/sub_tasks/15_user.yml

```diff
@@ -34,24 +33,8 @@
 
 - name: Generate SSH keys for accessing OCP cluster
   community.crypto.openssh_keypair:
+    comment: "cifmw generated key for ocp access"
     path: >-
       {{ cifmw_devscripts_artifacts_dir }}/{{ cifmw_devscripts_user }}_ed25519
     type: "ed25519"
   register: cifmw_devscripts_ssh_key
-
-- name: Enable password-less access
```

Comment:
oorr.. ok. you removed that here - so it shouldn't show up anymore. Any reason you added it in the first place?

Reply:
Initially, I wanted to create a separate user for deploying OCP. Then it became an overweight ... hence chopped it out :)